packages/ui/components/overlays/src/OverlayController.js

```diff
@@ -1121,6 +1121,15 @@ export class OverlayController extends EventTarget {
     if (this.manager) {
       this.manager.disableTrapsKeyboardFocusForAll();
     }
+
+    const isContentShadowHost = Boolean(this.contentNode.shadowRoot);
+    if (isContentShadowHost) {
+      // eslint-disable-next-line no-console
+      console.warn(
```

Comment:
So the fix is just a warning?

Reply:
Yes. Say we would be updating this in ing-web, we would be instructed to change it to a parent node in shadow dom that projects it (an error would be too breaking).

And that was exactly the solution for ing-web